frontend/src/App.vue

```diff
@@ -1,8 +1,24 @@
 <template lang="pug">
-  div(id="app")
-    router-view
+  v-app(style="background-color: white")
+    PageHeader
+    v-content
+      v-container(fluid :fill-height="$route.meta.fillHeight")
+        router-view
+    PageFooter
 </template>
 
+<script>
+import PageHeader from "@/components/Header.vue";
+import PageFooter from "@/components/Footer.vue";
+
+export default {
+  name: "App",
+  components: {
+    PageHeader,
+    PageFooter
+  }
+};
+</script>
 <style>
 #app {
```

Comment:
Is this of any use?

Reply:
Header and footer are to be shown in every page so making it global